test/release/preload-tiles.html

```diff
@@ -0,0 +1 @@
+../../debug/preload-tiles.html
```

Comment:
👍 One last step is to also reflect it in the index entry, such as https://github.com/mapbox/mapbox-gl-js/blob/main/test/release/index.js#L119_L122